go/libkb/saltpack.go

```diff
@@ -138,8 +138,6 @@ func BoxPublicKeyToKeybaseKID(k saltpack.BoxPublicKey) (ret keybase1.KID) {
 }
 
 func checkSaltpackBrand(b string) error {
-	if b != KeybaseSaltpackBrand {
-		return KeybaseSaltpackError{}
-	}
```

Comment:
I feel like as this is the first saltpack implementation released into the wild, that it should accept any brand.

If I write a version in haskell and want to test that it works, I would use the keybase CLI to check the messages I generated.  And with this in place, I would have to use KEYBASE brand...

Perhaps decrypt, verify could display the brand?